File: codeforces/34.i_wanna_be_the_guy/iwbtg.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int n, px, py;
  bool status = true;
  cin >> n;
  cin >> px;
  unsigned short int *x_levels = new unsigned short int[px];
  for (unsigned short int i = 0; i < px; i++) {
    cin >> x_levels[i];
  }
  cin >> py;
  unsigned short int *y_levels = new unsigned short int[py];
  for (unsigned short int i = 0; i < py; i++) {
    cin >> y_levels[i];
  }
  unsigned short int i = 1;
  unsigned short int counter = 0;
  while (i <= n && status) {
    status = false;
    for (unsigned short int j = 0; j < px; j++) {
      if (x_levels[j] == i) {
        status = true;
        counter++;
        break;
      }
    }
    for (unsigned short int j = 0; j < py && !status; j++) {
      if (y_levels[j] == i) {
        status = true;
        counter++;
        break;
      }
    }
    i++;
  }
  if (n == counter) {
    cout << "I become the guy." << endl;
  }
  else {
    cout << "Oh, my keyboard!" << endl;
  }
  delete[] x_levels;
  delete[] y_levels;
  return 0;
}

File: codeforces/34.i_wanna_be_the_guy_[469A]/iwbtg.py
# Author: José Rodolfo (jric2002)
n = int(input())
p = 0
levels = []
input_data = list(map(int, input().split(" ")))
p += input_data[0]
levels.extend(input_data[1:])
input_data = list(map(int, input().split(" ")))
p += input_data[0]
levels.extend(input_data[1:])
status = True
counter = 0
i = 1
while ((i <= n) and status):
  status = False
  for j in range(0, p):
    if (levels[j] == i):
      status = True
      counter += 1
      break
  i += 1
if (n == counter):
  print("I become the guy.")
else:
  print("Oh, my keyboard!")